adoc-mode.el

```diff
@@ -301,6 +301,33 @@ Also used to delimit the scan for the end delimiter."
   :group 'adoc
   :package-version '(adoc-mode . "0.8.0"))
 
+(defcustom adoc-max-image-size nil
+  "Maximum width and height for displayed inline images.
+This variable may be nil or a cons cell (MAX-WIDTH . MAX-HEIGHT).
+When nil, use the actual size.  Otherwise, use ImageMagick to
+resize larger images to be of the given maximum dimensions.  This
+requires Emacs to be built with ImageMagick support."
+  :group 'adoc
+  :package-version '(markdown-mode . "0.8.0")
```

Comment:
It should say `adoc-mode` here. :-) And I guess the version will be 0.9.

Reply:
I don't get the integration errors locally. Do you know what is happening over there on the server? I already tried to require `mouse` and `image` in the case that the server does not have a graphical environment and does not load these automatically. Currently, I can only test with Emacs 29.1.

Reply:
> It should say `adoc-mode` here. :-) And I guess the version will be 0.9.

Done.

Reply:
> @bbatsov I don't get the integration errors locally. Do you know what is happening over there on the server? I already tried to require `mouse` and `image` in the case that the server does not have a graphical environment and does not load these automatically. Currently, I can only test with Emacs 29.1.

Perhaps Emacs that use for the CI is not compiled with support for images? I think even in the terminal environment that's some optional functionality that probably doesn't make sense for most Emacs packages.